search/elastic.py

```diff
@@ -215,15 +278,12 @@ def log_indexing_error(cls, indexing_errors):
         """
         Logs indexing errors and raises a general ElasticSearch Exception
         """
-        raise exceptions.ElasticsearchException(', '.join(
-            [str(indexing_error) for indexing_error in indexing_errors]
-        ))
+        raise exceptions.ElasticsearchException(', '.join(list(map(str, indexing_errors))))
```

Comment:
I think this one will be even better `raise exceptions.ElasticsearchException(', '.join(map(str, indexing_errors)))`
I cut out `list`

Reply:
Done